test/shopify_app/controller_concerns/webhook_verification_test.rb

```diff
@@ -27,7 +38,21 @@ class WebhookVerificationTest < ActionController::TestCase
       params = { foo: 'anything' }
       digest = OpenSSL::Digest.new('sha256')
       secret = ShopifyApp.configuration.secret
-      valid_hmac = Base64.encode64(OpenSSL::HMAC.digest(digest, secret, params.to_query)).strip
+      valid_hmac = Base64.strict_encode64(OpenSSL::HMAC.digest(digest, secret, params.to_query))
```

Comment:
Just a suggestion, but I feel like this should be a helper method since you use it in the code and in multiple tests.

Reply:
I'm actually going to change the tests to assert the right result, instead of using code similar to the method implementation. 👍